app/src/main/java/com/mybus/adapter/BusResultViewAdapter.java

```diff
@@ -47,23 +47,20 @@ public void onBindViewHolder(BusResultViewHolder holder, int pos) {
         BusRouteResult routeResult = mDataset.get(pos);
         List<BusRoute> routes = routeResult.getBusRoutes();
         if (routes != null && !routes.isEmpty()) {
+            BusRoute firstBusRoute = routes.get(0);
+            holder.mStartAddress.setText(firstBusRoute.getStartBusStopStreetName() + " " + firstBusRoute.getStartBusStopStreetNumber());
```

Comment:
In this case, please use String.format, e.g:
`holder.mStartAddress.setText(String.format("%s` %s", firstBusRoute.getStartBusStopStreetName(), firstBusRoute.getStartBusStopStreetNumber()));`